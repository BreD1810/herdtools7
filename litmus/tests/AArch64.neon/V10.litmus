AArch64 V10
(* Tests load / store immediate, with offset *)
{
int64_t x; int8_t y;
0:X0=x; 0:X2=y;
}

 P0                 ;
 MOV X1,#1          ;
 STR X1, [X0, #8]   ;
 LDR B0, [X0, #8]   ;
 STR B0, [X2]       ;
 forall (y=1)
