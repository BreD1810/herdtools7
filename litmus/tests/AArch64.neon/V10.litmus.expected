Test V10 Required
Histogram (1 states)
1000000:>y=1;
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (y=1) is validated
Hash=ce2071645ccb10acb32fc1be0f489d5d
Observation V10 Always 1000000 0
